regression-test/suites/query_p0/sql_functions/string_functions/test_trim_in.groovy

```diff
@@ -0,0 +1,153 @@
+    order_qt_6 "SELECT ltrim_in(' hello world ');"
+    order_qt_7 "SELECT ltrim_in(CAST('' AS CHAR(20)));"
+    order_qt_8 "SELECT ltrim_in(CAST('  hello  ' AS CHAR(9)));"
+    order_qt_9 "SELECT ltrim_in(CAST('  hello' AS CHAR(7)));"
+    order_qt_10 "SELECT ltrim_in(CAST('hello  ' AS CHAR(7)));"
+    order_qt_11 "SELECT ltrim_in(CAST(' hello world ' AS CHAR(13)));"
+    order_qt_12 "SELECT rtrim_in('');"
+    order_qt_13 "SELECT rtrim_in('   ');"
+    order_qt_14 "SELECT rtrim_in('  hello  ');"
+    order_qt_15 "SELECT rtrim_in('  hello');"
+    order_qt_16 "SELECT rtrim_in('hello  ');"
+    order_qt_17 "SELECT rtrim_in(' hello world ');"
+    order_qt_18 "SELECT rtrim_in(CAST('' AS CHAR(20)));"
+    order_qt_19 "SELECT rtrim_in(CAST('  hello  ' AS CHAR(9)));"
+    order_qt_20 "SELECT rtrim_in(CAST('  hello' AS CHAR(7)));"
+    order_qt_21 "SELECT rtrim_in(CAST('hello  ' AS CHAR(7)));"
+    order_qt_22 "SELECT rtrim_in(CAST(' hello world ' AS CHAR(13)));"
+    order_qt_23 "SELECT ltrim_in('', '');"
+    order_qt_24 "SELECT ltrim_in('   ', '');"
+    order_qt_25 "SELECT ltrim_in('  hello  ', '');"
+    order_qt_26 "SELECT ltrim_in('  hello  ', ' ');"
+    order_qt_27 "SELECT ltrim_in('  hello  ', ' ');"
+    order_qt_28 "SELECT ltrim_in('  hello  ', 'he ');"
+    order_qt_29 "SELECT ltrim_in('  hello', ' ');"
+    order_qt_30 "SELECT ltrim_in('  hello', 'e h');"
+    order_qt_31 "SELECT ltrim_in('hello  ', 'l');"
+    order_qt_32 "SELECT ltrim_in(' hello world ', ' ');"
+    order_qt_33 "SELECT ltrim_in(' hello world ', ' eh');"
+    order_qt_34 "SELECT ltrim_in(' hello world ', ' ehlowrd');"
+    order_qt_35 "SELECT ltrim_in(' hello world ', ' x');"
+    order_qt_36 "SELECT ltrim_in(CAST('' AS CHAR(1)), '');"
+    order_qt_37 "SELECT ltrim_in(CAST('   ' AS CHAR(3)), '');"
+    order_qt_38 "SELECT ltrim_in(CAST('  hello  ' AS CHAR(9)), '');"
+    order_qt_39 "SELECT ltrim_in(CAST('  hello  ' AS CHAR(9)), ' ');"
+    order_qt_40 "SELECT ltrim_in(CAST('  hello  ' AS CHAR(9)), 'he ');"
+    order_qt_41 "SELECT ltrim_in(CAST('  hello' AS CHAR(7)), ' ');"
+    order_qt_42 "SELECT ltrim_in(CAST('  hello' AS CHAR(7)), 'e h');"
+    order_qt_43 "SELECT ltrim_in(CAST('hello  ' AS CHAR(7)), 'l');"
+    order_qt_44 "SELECT ltrim_in(CAST(' hello world ' AS CHAR(13)), ' ');"
+    order_qt_45 "SELECT ltrim_in(CAST(' hello world ' AS CHAR(13)), ' eh');"
+    order_qt_46 "SELECT ltrim_in(CAST(' hello world ' AS CHAR(13)), ' ehlowrd');"
+    order_qt_47 "SELECT ltrim_in(CAST(' hello world ' AS CHAR(13)), ' x');"
+    order_qt_48 "SELECT rtrim_in('', '');"
+    order_qt_49 "SELECT rtrim_in('   ', '');"
+    order_qt_50 "SELECT rtrim_in('  hello  ', '');"
+    order_qt_51 "SELECT rtrim_in('  hello  ', ' ');"
+    order_qt_52 "SELECT rtrim_in('  hello  ', 'lo ');"
+    order_qt_53 "SELECT rtrim_in('hello  ', ' ');"
+    order_qt_54 "SELECT rtrim_in('hello  ', 'l o');"
+    order_qt_55 "SELECT rtrim_in('hello  ', 'l');"
```

Comment:
add more test
1. create a table and then insert the values to a table, and use select func(col) from table
2. the column in the table, could be a nullable column and not null column.

Reply:
done